cmd/vulcan-zap/main.go

```diff
@@ -113,21 +117,22 @@ func main() {
 		}
 
 		for {
-			time.Sleep(1 * time.Second)
+			time.Sleep(10 * time.Second)
 			resp, err := client.Spider().Status(scanid)
 			if err != nil {
-				return fmt.Errorf("error getting the status of the scan: %v", err)
+				return fmt.Errorf("error getting the status of the spider: %v", err)
```

Comment:
Not a big deal, because we are not going to unwrap this error anway, but I here I would use this:
fmt.Errorf("error getting the status of the spider: %w", err)

Reply:
Ooooh, true, I did not modify the errors and they were written before wrapping was implemented in Go. I think it's a good moment to use the proper format string, so I will change it. Good catch :+1:

Reply:
Done!